sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/Transforms.java

```diff
@@ -0,0 +1,214 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.formrecognizer;
+
+import com.azure.ai.formrecognizer.implementation.models.AnalyzeResult;
+import com.azure.ai.formrecognizer.implementation.models.DocumentResult;
+import com.azure.ai.formrecognizer.implementation.models.ReadResult;
+import com.azure.ai.formrecognizer.implementation.models.TextLine;
+import com.azure.ai.formrecognizer.implementation.models.TextWord;
+import com.azure.ai.formrecognizer.models.BoundingBox;
+import com.azure.ai.formrecognizer.models.DimensionUnit;
+import com.azure.ai.formrecognizer.models.Element;
+import com.azure.ai.formrecognizer.models.ExtractedReceipt;
+import com.azure.ai.formrecognizer.models.FieldValue;
+import com.azure.ai.formrecognizer.models.FloatValue;
+import com.azure.ai.formrecognizer.models.IntegerValue;
+import com.azure.ai.formrecognizer.models.LineElement;
+import com.azure.ai.formrecognizer.models.PageMetadata;
+import com.azure.ai.formrecognizer.models.Point;
+import com.azure.ai.formrecognizer.models.ReceiptItem;
+import com.azure.ai.formrecognizer.models.ReceiptType;
+import com.azure.ai.formrecognizer.models.StringValue;
+import com.azure.ai.formrecognizer.models.TextLanguage;
+import com.azure.ai.formrecognizer.models.WordElement;
+import com.azure.core.util.IterableStream;
+import com.azure.core.util.logging.ClientLogger;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Pattern;
+
+/**
+ * Helper class to convert service level models to SDK exposed models.
+ */
+final class Transforms {
```

Comment:
Mind adding comments into the helper methods and in the code to explain the transforms being applied.